ext/lib/site/anchor/index.js

```diff
@@ -7,7 +7,12 @@ export default class Anchor extends Component {
   static anchors = {}
 
   static goTo (id) {
-    if (this.anchors[id]) {
+    if (this.anchors['container']) {
```

Comment:
`this.anchors['container']` va a ser verdadero mientras se haya instanciado el componente Anchor con ese id, por ende siempre va a entrar a ese if aunque se llame goTo con otro id.
Deberías chequear si `id === 'container'`